bin/dqcmd

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+usage() {
+    echo "$(tput bold)dqcmd$(tput sgr0)
+
+    A Rofi/dmenu interface to qcmd. Excepts all arguments o qcmd (see below).
+    "
+    qcmd -h | sed "s/qcmd/dqcmd/"
+    exit
+}
+
+case $1 in
+    -h|--help) usage ;;
+esac
+
+action=$(qcmd $@)
+
+# Path to menu application
+if [[ -n $(command -v rofi) ]]; then
+    menu="$(command -v rofi) -dmenu -columns 1"
+    global_mesg="Alt-1 	Prompt for args and show function help (if -f is present)
..   	Go back to menu.
C-u  	Clear input
Esc  	Exit"
+    action=$(echo -e "$action" | $menu -mesg "$global_mesg") # For rofi
+elif [[ -n $(command -v dmenu) ]]; then
+    menu="$(command -v dmenu)"
+    action=$(echo -e "$action" | $menu) # For dmenu
+else
```

Comment:
What if user has both and wants to use dmenu rather than rofi?

Reply:
I added additional argument to dqcmd to enforce dmenu instead of rofi. As the default I left rofi as it gives better user experience in this case.